Hardcode/vision_image.py

```diff
@@ -0,0 +1,68 @@
+''' 
+******************** Pipeline driving code ******************
+Run:
+
+python vision_image.py \
+	-i ../../datasets/RoboMasterLabelledImagesSet1/image-550.jpg \
+	--save ./output/
+
+'''
+
+from resource import Mask
+from resource import Shape
+from resource import MatchLEDs
+from resource import Target
+import argparse
+import imutils
+import cv2
+
+# construct the argument parser and parse the arguments
+ap = argparse.ArgumentParser()
+ap.add_argument("-i", "--image", required=True,
+	help="path to the input image")
+ap.add_argument("--save", required=True, 
+	help="Path to saved image directory")
+args = vars(ap.parse_args())
```

Comment:
We should encapsulate all this in an 
`if __name__ == "__main__"`.

That way, if we eventually import some part of this in some other script, this code won't be run.